INSTRUMENTS/SERIAL/SERIAL_TIMESERIES/SERIAL_TIMESERIES.py

```diff
@@ -41,10 +41,11 @@ def SERIAL_TIMESERIES(dc_inputs, params):
     num_readings * record_period is roughly the run length in seconds.
```

Comment:
Could you please make sure that all parameter names match with the manifest file?